csharp/README.md

```diff
@@ -1,14 +1,29 @@
 # CSharp (C#)
 
+## Queries
+
+<!-- AUTOMATION-QUERIES -->
```

Comment:
Added <!-- AUTOMATION-QUERIES --> so that we get the new queries added to docs

Reply:
Looks so.